docs/reference/getting-started.asciidoc

```diff
@@ -31,156 +31,158 @@ trial of Elasticsearch Service] in the cloud.
-[2018-09-13T12:20:05,221][WARN ][o.e.b.BootstrapChecks    ] [localhost.localdomain] max file descriptors [4096] for elasticsearch process is too low, increase to at least [65535]
-[2018-09-13T12:20:05,221][WARN ][o.e.b.BootstrapChecks    ] [localhost.localdomain] max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]
-[2018-09-13T12:20:08,355][INFO ][o.e.c.s.MasterService    ] [localhost.localdomain] elected-as-master ([0] nodes joined)[, ], reason: master node changed {previous [], current [{localhost.localdomain}{B0aEHNagTiWx7SYj-l4NTw}{hzsQz6CVQMCTpMCVLM4IHg}{127.0.0.1}{127.0.0.1:9300}{testattr=test}]}
-[2018-09-13T12:20:08,360][INFO ][o.e.c.s.ClusterApplierService] [localhost.localdomain] master node changed {previous [], current [{localhost.localdomain}{B0aEHNagTiWx7SYj-l4NTw}{hzsQz6CVQMCTpMCVLM4IHg}{127.0.0.1}{127.0.0.1:9300}{testattr=test}]}, reason: apply cluster state (from master [master {localhost.localdomain}{B0aEHNagTiWx7SYj-l4NTw}{hzsQz6CVQMCTpMCVLM4IHg}{127.0.0.1}{127.0.0.1:9300}{testattr=test} committed version [1] source [elected-as-master ([0] nodes joined)[, ]]])
-[2018-09-13T12:20:08,384][INFO ][o.e.h.n.Netty4HttpServerTransport] [localhost.localdomain] publish_address {127.0.0.1:9200}, bound_addresses {[::1]:9200}, {127.0.0.1:9200}
-[2018-09-13T12:20:08,384][INFO ][o.e.n.Node               ] [localhost.localdomain] started
-
---------------------------------------------------
-
-Without going too much into detail, we can see that our node named "localhost.localdomain" has started and elected itself as a master in a single cluster. Don't worry yet at the moment what master means. The main thing that is important here is that we have started one node within one cluster.
-
-As mentioned previously, we can override either the cluster or node name. This can be done from the command line when starting Elasticsearch as follows:
-
-[source,sh]
+// CONSOLE
++
+The response should indicate that the status of the _elasticsearch_ cluster
+is _green_ and it has three nodes:
++
+[source,txt]
 --------------------------------------------------
-./elasticsearch -Ecluster.name=my_cluster_name -Enode.name=my_node_name
+epoch      timestamp cluster       status node.total node.data shards pri relo init unassign pending_tasks max_task_wait_time active_shards_percent
+1565052807 00:53:27  elasticsearch green           3         3      6   3    0    0        0             0                  -                100.0%
 --------------------------------------------------
+// TESTRESPONSE[s/1565052807 00:53:27  elasticsearch/\\d+ \\d+:\\d+:\\d+ integTest/]
+// TESTRESPONSE[s/3         3      6   3/\\d+         \\d+      \\d+   \\d+/]
+// TESTRESPONSE[s/0             0                  -/0             \\d+                  -/]
+// TESTRESPONSE[non_json]
++
+NOTE: The cluster status will remain yellow if you are only running a single
```

Comment:
Not sure if this note is needed since we're explicitly creating a three-node cluster.

Reply:
I opted to leave it. This isn't strictly necessary...if folks follow the instructions. But, it's pretty common to wonder why the status of a freshly-started cluster is yellow.